client/src/Components/Search/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import SearchButton from './SearchButton';
+import './style.css';
+
+interface SearchByLocationProps {
```

Comment:
this interface repeated with different name 
you can export it and user from anther pace